### ISSUE TITLE
'matplotlib' GUI not capturing keyboard focus on Mac OSX 10.10.3

### ISSUE THREAD

**Issue by cgddrd** [cgddrd/CS39440-major-project#136]
'matplotlib' Python plotting library fails to capture any keyboard focus while running on Mac OSX 10.10.3 with Python 2.7 (Homebrew)

Similar issue to: https://github.com/matplotlib/matplotlib/issues/665/
